tendrl/integrations/ceph/sds_sync/__init__.py

```diff
@@ -0,0 +1,108 @@
+import gevent
+import json
+import os
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message, ExceptionMessage
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons import sds_sync
+from tendrl.node_agent.node_sync import disk_sync
+
+
+class CephIntegrtaionsSyncThread(sds_sync.StateSyncThread):
+    def _run(self):
+        Event(
+            Message(
+                priority="info",
```

Comment:
this should be debug, this isnt useful info for the user, this is more for the devs/support